package.json

```diff
@@ -32,62 +32,62 @@
     "check:everything": "ts-node ./.scripts/checkEverything.ts",
     "latest": "ts-node ./.scripts/latest.ts",
     "local": "ts-node ./.scripts/local.ts",
-    "install-client-eventhubs-client": "cd packages/@azure/eventhubs/client && npm install",
-    "install-client-eventhubs-processor": "cd packages/@azure/eventhubs/processor && npm install",
-    "install-client-keyvault": "cd packages/@azure/keyvault && npm install",
+    "install-client-eventhubs-client": "cd sdk/eventhubs/event-hubs && npm install",
```

Comment:
I know you didn't name this but in order to help drive consistency can we please name this eventhub instead of eventhubs? That will match the api spec repo (https://github.com/Azure/azure-rest-api-specs/tree/master/specification/eventhub) and make sense that we don't make these plural.

Reply:
Either name sounds good to me, just another data point: the service is named `Event Hubs` in our doc site (https://azure.microsoft.com/en-us/services/event-hubs/) though

Reply:
Yeah I can update that

Reply:
The package name is `@azure/event-hubs`, I would expect the folder to be `event-hubs` as well.
Now that we are renaming `client` and `processor` to `event-hubs` and `event-processor-host`, I would skip the `eventhubs` folder altogether and have `sdk/event-hubs` and `sdk/event-processor-host`.

Reply:
The format we are going for is "sdk\<service name>\<package name>", so the question really is what is the service name? The auto-rest call it `eventhub` and so does the mgmt package `arm-eventhub` so I was trying to be consistent with those. However there is definitely a number of inconsistencies around using an `s` or not, just from browsing through the docs and the azure portal I see a a mix of both various services, for example in docs it is `Key Vault` but in the portal it is `Key Vaults`. My preference would be to use the singular form for the service name. 

>The package name is @azure/event-hubs, I would expect the folder to be event-hubs as well.

The package name will be the sub-folder under the service name.

Reply:
That makes sense, thanks @weshaggard